README.md

```diff
@@ -103,7 +103,7 @@ Trabalhos
  - [P-corridas2019](https://github.com/ISEC-estudantes/P-corridas2019) feito por [OnikenX](https://github.com/OnikenX) em 2019
  - [P-virus2020](https://github.com/JOSEALM3IDA/isec-material/tree/main/PROG-2020/progtrabalho01) feito por [JOSEALM3IDA](https://github.com/JOSEALM3IDA) em 2020
  - [Covid-Prog](https://github.com/BrunoTeixeira1996/Apontamentos-ISEC/tree/master/1%C2%BAAno/2%C2%BASemestre/Programa%C3%A7%C3%A3o/TP) feito por [BrunoTeixeira1996](https://github.com/BrunoTeixeira1996) em 2020
- - [Propagacao_Virus](https://github.com//TrabalhoP_Simulacao_Propagacao_Virus) feito por [ (Diogo Pascoal)](https://github.com/) em 2020
+ - [Propagacao_Virus](https://github.com//TrabalhoP_Simulacao_Propagacao_Virus) feito por [ (Diogo Pascoal)](https://github.com/sirNugg3ts) em 2020
```

Comment:
continua inválido https://github.com//TrabalhoP_Simulacao_Propagacao_Virus

Reply:
ah dude qual e a cena do github, isso foi copy paste URL, how dafuq did that happen

Reply:
dps de fazer paste vai ao preview para verificar, n sei o porque disso

Reply:
ja corrigi os que tinham o //